### ISSUE TITLE
What is going wrong here?

### ISSUE THREAD

**Issue by TiagoTiago** [chaintip/chaintip-reddit#6]
https://www.reddit.com/r/btc/comments/cfkxk4/a_newbie_that_wants_to_find_the_best_wallet_but_i/euapmzx/?depth=16

Does Chaintip not support the CashAddress format? Why not even a notification about an unsupported format?

**Comment by chaintip** [comment]
Yeah, I think you're right about it being slightly ambiguous as many people have tried to claim like that: https://www.reddit.com/r/btc/comments/cfkxk4/a_newbie_that_wants_to_find_the_best_wallet_but_i/eubjo1j/

It does support the new format :)

This is what is says atm when you click on that:

![Screenshot 2019-11-16 at 20 19 49](https://user-images.githubusercontent.com/32135872/68998772-8829e480-08ae-11ea-9e7a-9110279dacb3.png)

How do you think I can make it less ambiguous?

**Comment by testtip** [comment]
Fixed this now btw... thanks @TiagoTiago @chaintip 

https://www.chaintip.org/#claim

**Comment by chaintip** [comment]
[chaintip](http://www.chaintip.org) has [returned](https://explorer.bitcoin.com/bch/address/bitcoincash:qqh0ld4fgvxhd0pccu2hgtef3fde8u02x583nfhcwc) the unclaimed tip of `0.01238402 BCH`| `~   2.66 USD` to @testtip.
